server/service/GithubService.js

```diff
@@ -23,14 +25,12 @@ const PATHS = {
 export default class GithubService {
 
   getOptions(method, path, body, accessToken) {
-    let url = nconf.get('GITHUB_URL') + path
-
     return {
       json: true,
       method: method,
-      url,
+      url: GITHUB_API_ADDR + path,
       headers: {
-        'User-Agent': 'ZAPPR/1.0 (+https://zappr.hackweek.zalan.do)',
+        'User-Agent': `ZAPPR/1.0 (+${HOST_ADDR})`,
```

Comment:
Ok. Turns out the version is [optional](https://tools.ietf.org/html/rfc7231#section-5.5.3) so I'm going to use the first suggestion.